specs/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,235 @@
+ElementBackgroundGrid
+===
+
+# Background
+
+The `ElementBackgroundGrid` API in this spec is a Xaml Grid that lets you "draw" its background by
```

Comment:
I understand the derivation of this suggested name but off the cuff the name makes it seem way more interesting than it is. It is a very purpose-built primitive that I don't expect to see much other use for, so I would like to give it a name that makes it seem less attractive. As-is (without reading this description) it makes it sound like something I might want to use as the background-most Grid for any Element (i.e. Control) I might be wanting to build.

I'm not saying the name I originally gave it was better, but I would hope that we could make the name less generic-sounding to make it sound a little less generally useful.

This description and name also omits the ColorEffect operation that the primitive performs, which is a key aspect to its purpose-built nature.

Reply:
Why not make it a more "interesting" control for others to use.  There is talk about a ButtonGroup/SegmentedControl where I suggested this primitive control be used for that control.
https://github.com/microsoft/microsoft-ui-xaml/issues/2310#issuecomment-799954635

Reply:
I think the current name is too misleading. The name and description made me think it did something completely different from what it actually does. (This confusion will become evident in some of my comments.)

Reply:
Discussion: Does this need to be a Grid? Border is sealed. Does it need any child(ren) though? We could potentially derive from FE.

This *only* works where it overlaps the source element. This really is only to be used as an overlay. This is a primitive that's only designed to be used as part of a control template. If we want at some point we could create a general-purpose control that uses this in its control template.

Colorizier/Monochromizer?
[BitmapIcon.ShowAsMonochromeProperty](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon.ShowAsMonochromeProperty)

**Naming:**
Overlay is nice
ElementBackground is nice?
Presenter is nice
OverlayElementBackgroundPresenter
ElementBackgroundOverlayPresenter
OverlayEffectPresenter
ElementOverlayMonochromePresenter (Source, ReplacementColor)
Colorizer/Monochromizer
ElementMonochromeOverlayPresenter
MonochromaticOverlayPresenter

Recommend:
Primitives namespace
MonochromaticOverlayPresenter : FrameworkElement